aea/components/base.py

```diff
@@ -131,9 +130,6 @@ def load_aea_package(configuration: ComponentConfiguration) -> None:
     prefix_author = prefix_root + f".{configuration.author}"
     prefix_pkg_type = prefix_author + f".{configuration.component_type.to_plural()}"
     prefix_pkg = prefix_pkg_type + f".{configuration.name}"
-    sys.modules[prefix_root] = types.ModuleType(prefix_root)
-    sys.modules[prefix_author] = types.ModuleType(prefix_author)
-    sys.modules[prefix_pkg_type] = types.ModuleType(prefix_pkg_type)
```

Comment:
@marcofavorito why was this added in the first place?

Reply:
to allow imports for prefixes of import paths. And to improve the error message in case of wrong prefix path. (e.g. existing author but missing package)

Reply:
Where do we need `imports for prefixes of import paths.`?

Re error message, we then need to improve the error logging there to ensure this doesn't get worse as a result of this PR.

Reply:
done